docs/spark-api/asciidoc/fr/spark-api.ad

```diff
@@ -0,0 +1,493 @@
+= SPARK API
+:subtitle: Data Historian 1.0
+:doctype: book
+:title-logo-image: image:logo.png[pdfwidth=3.5in,align=center]
+:author: (C)Hurence
+:email: contact@hurence.com
+:revnumber: v1.0
+:revdate: 18.02.2020
+:revremark: First draft
+:toc:
+:toclevels: 4
+
+== Introduction
+L'api-spark est une api qui sert à lire les données à l'aide de spark, les chunkifier et les injecter finalement dans Solr.
+
+
+Il y a deux façons pour utiliser cette api :
+
+=== Spark Shell
+Pour installer apache spark vous pouvez télécharger cette archive :
+https://archive.apache.org/dist/spark/spark-2.3.4/spark-2.3.4-bin-without-hadoop.tgz[spark-2.3.4-bin-without-hadoop.tgz]
+
+Les commandes suivantes vous permettront de faire une installation locale.
+
+----
+# installer Apache Spark 2.3.4 and unpack it
+cd $HDH_HOME
+wget https://archive.apache.org/dist/spark/spark-2.3.4/spark-2.3.4-bin-without-
+hadoop.tgz
+tar -xvf spark-2.3.4-bin-without-hadoop.tgz
+rm spark-2.3.4-bin-without-hadoop.tgz
+
+# add two additional jars to spark to handle our framework
+wget -O spark-solr-3.6.6-shaded.jar https://search.maven.org/remotecontent?filepath=com/lucidworks/spark/spark-solr/3.6.6/spark-solr-3.6.6-shaded.jar
+mv spark-solr-3.6.6-shaded.jar $HDH_HOME/spark-2.3.4-bin-without-hadoop/jars/
+cp $HDH_HOME/historian-1.3.4-SNAPSHOT/lib/loader-1.3.4-SNAPSHOT.jar $HDH_HOME/spark-2.3.4-bin-without-hadoop/jars/
+----
```

Comment:
Ne pas utiliser HDH_HOME. Ni mentionner historian-1.3.4-SNAPSHOT !!!
C'est une doc officiel de la version 1.3.5 en plus c'est choquant que tu mentionne une version snapshot ici....

Bref expliques juste qu'il faut rajouter la jar loader et qu'elle est disponible dans le tar.gz de la release  https://github.com/Hurence/historian/releases/download/v1.3.5/historian-1.3.5-bin.tgz ou bien si installer via le script au niveau de l'install dans hitoriean-1.3.5/path/to/jar et pas snapshot !